operator/v1/types_network.go

```diff
@@ -349,6 +353,35 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type PolicyAuditConfig struct {
+	// RateLimit is the approximate maximum number of messages to generate per-second per-node. If
+	// unset the default of 20 msg/sec is used.
+	// +optional
+	RateLimit *uint32 `json:"rateLimit,omitempty"`
+
+	// MaxFilesSize is the max size an ACL_audit log file is allowed to reach before rotation occurs
+	// Default is 50MB
+	// +optional
+	MaxFileSize *uint32 `json:"maxFileSize,omitempty"`
+
+	// Messages are output in syslog format. Destination is the destination for policy log messages.
+	// Regardless of this config logs will always be dumped to ovn at /var/log/ovn/ however
+	// you may also configure additional output as follows.
+	// Messages are output in syslog format.
+	// Valid values are:
+	// - "libc" -> to use the libc syslog() function of the host node's journdald process
+	// - "udp:host:port" -> for sending syslog over UDP
+	// - "unix:file" -> for using the UNIX domain socket directly
+	// - "null" -> to discard all messages logged to syslog
```

Comment:
add validation for all of them (regex)

Reply:
Can't I perform validation for this field in the Cluster network operator @sttts? That's what I was planning on doing anyways.

Reply:
Added Regex Validation and Defaults